### PR TITLE
New layouts for indicators from 6.10.0

### PR DESCRIPTION
**Pull request by Ni-Knight**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-4713

## Description
In 6.10.0 we added s REG/EDIT tab to indicators so we can now edit which fields are shown.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0
- [x] 6.10.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/CommonTypes/IndicatorTypes/reputation-Onion_Address.json [hunk 1]
```diff
@@ -12,5 +12,6 @@
     "system": false,
     "updateAfter": 0,
     "version": -1,
-    "fromVersion": "5.0.0"
+    "fromVersion": "5.0.0",
+    "layout": "domainRepUnified"
```
Comment (altmannyarden) [inline Packs/CommonTypes/IndicatorTypes/reputation-Onion_Address.json:16]: Onion address - should have the domain layout?

Comment (Ni-Knight) [reply Packs/CommonTypes/IndicatorTypes/reputation-Onion_Address.json:16]: it is a domain, we don't have a layout for it. It's either that or no layout which is exactly the same (only seems like bad practice to me).

File: Packs/CommonTypes/Layouts/layoutscontainer-Windows_Registry_KeyV2.json [hunk 2]
```diff
@@ -0,0 +1,782 @@
+{
+    "fromVersion": "6.10.0",
+    "edit": {
+        "sections": [
```
Comment (altmannyarden) [inline Packs/CommonTypes/Layouts/layoutscontainer-Windows_Registry_KeyV2.json:4]: Layout is missing the grid from the "new form"
image

Comment (Ni-Knight) [reply Packs/CommonTypes/Layouts/layoutscontainer-Windows_Registry_KeyV2.json:4]: Updated

**Review by altmannyarden — APPROVED** [review]

File: Packs/CommonTypes/Layouts/layoutscontainer-Vulnerability.json [hunk 3]
```diff
@@ -4,6 +4,7 @@
     "name": "Vulnerability Incident",
     "version": -1,
     "fromVersion": "6.0.0",
+    "toVersion": "6.9.9",
```
Comment (altmannyarden) [inline Packs/CommonTypes/Layouts/layoutscontainer-Vulnerability.json:7]: As it's not indicator layout (but incident) let's revert this layout change

**Review by altmannyarden — CHANGES_REQUESTED** [review]
see my last comment

File: Packs/CommonTypes/Layouts/layoutscontainer-Vulnerability.json (hunk 3 above)
Comment (Ni-Knight) [reply Packs/CommonTypes/Layouts/layoutscontainer-Vulnerability.json]: Yep, that happens when manually editing so many files :)

File: Packs/CommonTypes/IndicatorFields/indicatorfield-Report_type.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    "neverSetAsRequired": false,
+    "isReadOnly": false,
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 2,
+    "hidden": false,
+    "openEnded": false,
+    "associatedToAll": true,
```
Comment (altmannyarden) [inline Packs/CommonTypes/IndicatorFields/indicatorfield-Report_type.json:21]: why is that` associatedToAll : tru`e and user ID field `associatedToAll : false`?

Comment (Ni-Knight) [reply Packs/CommonTypes/IndicatorFields/indicatorfield-Report_type.json:21]: switched it to `True`

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**